includes/classes/debug.php

```diff
@@ -296,8 +296,12 @@ public function do_debug_options_actions( $settings ) {
 	public function handle_stuck_statuses( $settings, $back_button ) {
 		global $wpdb;
 
-		$sql     = "SELECT `option_name` FROM `$wpdb->options` WHERE `option_name` LIKE ('gc_pull_item_%') OR `option_name` LIKE ('gc_push_item_%');";
-		$options = $wpdb->get_results( $sql );
+		$options = $wpdb->get_results(
+			$wpdb->prepare(
+				"SELECT `option_name` FROM %s WHERE `option_name` LIKE ('gc_pull_item_%%') OR `option_name` LIKE ('gc_push_item_%%');",
+				$wpdb->options
```

Comment:
Does this definitely work as I had problems with doing prepared statements and the table name, it never seemed to be able to find the table?

Reply:
Didn't work! Whacked an ignore in!